tezos-sig-gating-contracts/package.json

```diff
@@ -11,18 +13,38 @@
     "localdeploy": "ts-node scripts/manual/localDeployNftminter.ts",
     "localmint": "ts-node scripts/manual/localMintOffchain.ts",
     "localdisplay": "ts-node scripts/manual/localDisplayStorage.ts",
+    "sandbox-start": "./scripts/run-sandbox tezos-sig-gating-contracts",
     "build": "preconstruct build",
     "pre-build": "preconstruct build",
     "publish-sdk-pkg": "pnpm build && pnpm publish --no-git-checks --registry https://npm.pkg.github.com/",
-    "lint": "prettier --write .",
-    "lint:check": "prettier --check ."
+    "lint": "prettier --write src/**/**.ts",
+    "lint:check": "prettier --check src/**/**.ts",
+    "inject_compiled_contracts": "make install && make compile && cp compiled/gatednftminter_simple.json src/compiledcontracts/"
```

Comment:
And and btw you might want to run pnpm lint after copying files because they might not be linted in the original json, so if you don't lint, it will commit changes even if the json didn't change, only because of linting